apps/frontend-manage/src/components/interaction/AudienceInteraction.tsx

```diff
@@ -71,173 +73,212 @@ function AudienceInteraction({
-                  'Running Session',
-                  'Feedback Moderation Toggled',
-                  String(!isModerationEnabled),
-                ])
-              }}
-              label="Moderation aktivieren"
-              className={{
-                label: twMerge(!isAudienceInteractionActive && 'text-gray-400'),
-              }}
-            />
-          </div>
-        </div>
-
-        <div className="flex flex-col flex-wrap self-start w-full gap-4 md:flex-row print:hidden">
-          {isAudienceInteractionActive && (
-            <div className="order-3 md:order-1">
-              <a
-                href={`/sessions/${sessionId}/lecturer`}
-                rel="noopener noreferrer"
-                target="_blank"
-              >
-                <Button className={{ root: 'h-10 px-4' }}>
-                  <Button.Icon className={{ root: 'mr-1' }}>
-                    <FontAwesomeIcon icon={faUpRightFromSquare} />
-                  </Button.Icon>
-                  <Button.Label>Dozierendenansicht</Button.Label>
-                </Button>
-              </a>
-            </div>
-          )}
-        </div>
+    <div className="flex flex-col flex-wrap justify-between md:flex-row">
+      <div className="hidden print:block">
+        <h1>Session &quot;{sessionName}&quot; - Feedback-Channel</h1>
       </div>
 
-      {isAudienceInteractionActive && (
-        <div className="flex flex-col gap-4 md:flex-row md:flex-wrap">
-          <div className="flex flex-col flex-1 md:flex-row">
-            <div className="flex-1">
-              <FeedbackChannel
-                feedbacks={feedbacks}
-                handleDeleteFeedback={(feedbackId: number): void => {
-                  deleteFeedback({ variables: { id: feedbackId } })
-                  push(['trackEvent', 'Running Session', 'Feedback Deleted'])
-                }}
-                handleDeleteFeedbackResponse={(responseId: number) => {
-                  deleteFeedbackResponse({
-                    variables: { id: responseId },
-                  })
-                  push([
-                    'trackEvent',
-                    'Running Session',
-                    'Feedback Response Deleted',
-                  ])
-                }}
-                handlePinFeedback={(feedbackId: number, isPinned: boolean) => {
-                  pinFeedback({
-                    variables: { id: feedbackId, isPinned: isPinned },
+      <div
+        className={twMerge(
+          'flex flex-row justify-start w-full gap-2 md:gap-8 md:flex-row print:hidden'
+        )}
+      >
+        <div className="flex flex-col w-2/3 pr-5">
+          <div className="flex flex-row justify-between">
+            <div className="flex pr-5 text-2xl font-bold">Live Q&A</div>
+            <div className="flex flex-row gap-6">
+              <Switch
+                checked={isLiveQAEnabled}
+                onCheckedChange={(): void => {
+                  changeSessionSettings({
```

Comment:
Only because we turn of the liveQAChannel we shouldn't turn off moderation automatically with it, right? The button is already disabled, but why should we reset the button to our default? And the setting won't have an effect anyways as long as the QA Channel is disabled

Reply:
done ✅